bundles/org.openhab.binding.ahawastecollection/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="ahawastecollection"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="collectionSchedule">
+		<label>aha Waste Collection Schedule</label>
+		<description>aha Waste Collection Schedule from http://www.aha-region.de/abholtermine/abfuhrkalender</description>
+
+		<channels>
+			<channel id="generalWaste" typeId="collectionDateGeneralWaste"/>
+			<channel id="leightweightPackaging" typeId="collectionDateLeightweightPackaging"/>
+			<channel id="bioWaste" typeId="collectionDateBioWaste"/>
+			<channel id="paper" typeId="collectionDatePaper"/>
+		</channels>
+
+		<config-description>
+			<parameter name="commune" type="text" required="true">
+				<label>Commune</label>
+				<description>The selected commune, taken from the form field 'gemeinde'. </description>
+			</parameter>
+			<parameter name="street" type="text" required="true">
+				<label>Street</label>
+				<description>The selected street, taken from the form field 'strasse'. This value must look like
+					67269@Rosmarinweg+/+Kirchhorst@Kirchhorst</description>
+			</parameter>
+			<parameter name="houseNumber" type="text" required="true">
+				<label>House Number</label>
+				<description>The selected house number, taken from the form field 'hausnr'.</description>
+			</parameter>
+			<parameter name="houseNumberAddon" type="text" required="false">
+				<label>House Number Addon</label>
+				<description>The selected house number addon, taken from the form field 'hausnraddon'.</description>
+			</parameter>
+			<parameter name="collectionPlace" type="text" required="true">
+				<label>Collection Place</label>
+				<description>Form value for the collection place, taken from the form field `ladeort`. This value must look like
+					67269-0010+</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<!-- Channel type that represents the collection time of an waste type. -->
+	<channel-type id="collectionDateGeneralWaste">
+		<item-type>DateTime</item-type>
+		<label>collection day general waste</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions
```suggestion
		<label>General Waste</label>
```